Feat/46 custom prop name (#59)
thisdot/open-source: libs/route-config/README.md

  This text is only visible, if there is a 'show' tag in the route data's `routeTags` Array
</p>
<ng-template #noShowTag>
  <p>There is no 'show' tag in this route's config</p>
</ng-template>
```

---

#### RouteDataHasDirective (`*tdRouteDataHas`)

> In case you need to use a different route data's property to store the tags you can use `*tdRouteDataHas` directive. It works very similarly to `*tdRouteTag` directive but provides a way to use different property as a source of data.

Reviewer: `If you need to use a different route data property to store the tags, you can use the `*tdRouteDataHas` directive.` <-- you don't need the possessive `'s` there, don't start the sentence with `In case` and add a comma after `tags`

Reviewer: `similarly` --> `similar`

`different property` --> `different properties`

Reviewer: ```suggestion
If you need to use a different route data property to store the tags, you can use `*tdRouteDataHas` directive. It works very similar to `*tdRouteTag` directive but provides a way to use different properties as a source of data.
```

Author: this is resolved by the next suggestion

> In case you need to use a different route data's property to store the tags you can use `*tdRouteDataHas` directive. It works very similarly to `*tdRouteTag` directive but provides a way to use different property as a source of data.

Reviewer: ```suggestion
If you need to use a different route data property to store the tags, you can use the `*tdRouteDataHas` directive. It works very similar to `*tdRouteTag` directive but provides a way to use different properties as a source of data.
```

> Let's imagine the following router configuration:

Reviewer: ```suggestion
Let's take a look at the following router configuration:
```

```ts
@NgModule({
  declarations: [FirstRouteComponent, SecondRouteComponent],
  imports: [
    RouterModule.forRoot([
      {
        path: 'first',
        component: FirstRouteComponent,
        data: {
          customDataProperty: ['customShow'],
        },
      },
      {
        path: 'second',
        component: SecondRouteComponent,
      },
    ]),
  ],
  exports: [RouterModule],
})
export class AppModule {}
```

> Now to configure the directive to use this property we can use `tdRouteDataHasPropName` input to set the desired property name:

Reviewer: ```suggestion
Now to configure the directive to use the `customDataProperty` property we can use `tdRouteDataHasPropName` input to set the desired property name:
```
